.github/workflows/tests.yml

```diff
@@ -94,17 +94,6 @@ jobs:
     name: Package Unit Tests
     needs: should_run_tests
     runs-on: ubuntu-latest
-    services:
-      postgres:
-        image: postgres:12
-        env:
-          POSTGRES_USER: keystone5
-          POSTGRES_PASSWORD: k3yst0n3
-          POSTGRES_DB: test_db
-        ports:
-          - 5432:5432
-    env:
-      DATABASE_URL: 'postgres://keystone5:k3yst0n3@localhost:5432/test_db'
```

Comment:
This was only used for the ecommerce project tests which will now just always use SQLite so this isn't needed.